contributor_docs/project_wrapups/shantanuKaushik_gsoc_2021.md

```diff
@@ -0,0 +1,57 @@
+  # Adding to p5.js Friendly Error System
+
+#### By Shantanu Kaushik ([@Aloneduckling](https://github.com/Aloneduckling)) | GSoC 2021
+
+### Overview
+Throughout this summer, I worked on p5.js's Friendly Error System (FES in short) under the mentorship of Thales Grilo and Luis Morales-Navarro. The FES is a p5.js feature
+which helps beginners with common errors, transforming error messages into a simple form, and checking for common mistakes. 
+This project aimed to further improve FES by adding new features and adding more to an existing feature.
+
+The major goals of this project were:
+1.  Additions to FES's `fesErrorMonitor`.
+2.  Allowing FES to detect redeclaration of p5.js reserved variables and functions
+
+
+### Work
+#### Phase 1: Adding to FES's `fesErrorMonitor`
+
+`fesErrorMonitor` functionality of FES shows the error given by the browsers and its details in a simple form. This feature was introduced to FES in GSoC 2020 by Akshay Padte.
+
+My work was to extend this functionality to detect more errors which the user can encounter and show them in a simplified form.
```

Comment:
My work was to extend this functionality to detect further errors a user can encounter and show them in a simplified form.

Reply:
Why was this work important?